qiskit/circuit/library/n_local/real_amplitudes.py

```diff
@@ -115,19 +118,18 @@ class RealAmplitudes(TwoLocal):
 
     def __init__(
         self,
-        num_qubits: Optional[int] = None,
-        entanglement: Union[str, List[List[int]], Callable[[int], List[int]]] = "reverse_linear",
+        num_qubits: int | None = None,
+        entanglement: str | list[list[int]] | Callable[[int], list[int]] = "full",
```

Comment:
This is a change in default.

Reply:
yes, it's always better to do "reverse_linear" and not "full" since it provides the same unitary, but with O(n) CX gates instead of O(n^2)